### DISCUSSION TITLE
AGU 2024 abstract submission

### DISCUSSION THREAD

**Discussion by asteiker** [earthaccess-dev/earthaccess#774] [General]
Let's get the word out about earthaccess at the AGU24 meeting! 

@JessicaS11 and I brainstormed ideas at today's hackday:

[IN043 - Showcasing your Earth Data Products, Tools, and Services](https://agu.confex.com/agu/agu24/prelim.cgi/Session/225163)

- This session is looking for key strengths, limitations, usage, how to access, etc.
- Could be a great opportunity to advertise the value of earthaccess and promote its usage in the context of climate variables and disasters (highlighting key NASA Earthdata for this) per the session abstract.
- @JessicaS11 is also planning to submit on icepyx separately, especially given the Harmony transition, plus updates to cloud reads (in prep for version 7)

We can also go the software ecosystem angle: Open science success story with all different communities coming together to streamline multiple services into one. We described this problem space in our ROSES proposal. 
- Related to EOSDIS SE TIM session, focused on community development lessons learned: https://docs.google.com/document/d/1-KMGjyNo1VxjG3wMqAJJk6A-RyN7IIjdl45YFDpwXms/edit#heading=h.bhoeu8o0rx5d 
- What sessions are most relevant?
-- [ED034 - Reflections on Open Science: Sharing Stories, Progress, and Lessons Learned](https://agu.confex.com/agu/agu24/prelim.cgi/Session/228837)
-- Others??

**Comment by asteiker** [comment]
I created an abstract draft in the shared Openscapes AGU doc for IN043: https://docs.google.com/document/d/1RbuVA66X4Rl9MQhfYhZJh-RFFTM9uYGxtrox7KDwi8o/edit#heading=h.n4nwildqooyo

Tagging all maintainers: @andypbarrett @jrbourbeau @chuckwondo @MattF-NSIDC @danielfromearth @jhkennedy @betolink @JessicaS11 . Please signal in the doc whether you do _not_ wish to be included in the abstract submission, and please suggest others who I may have missed.

**Comment by asteiker** [comment]
I submitted the abstract yesterday evening: https://agu.confex.com/agu/agu24/prelim.cgi/Paper/1706883. @jrbourbeau You were not listed in the AGU directory so I could not include you - but please let me know if I somehow missed your name or if you would still like to be included (they extended the deadline through today). Thanks everyone for your input!

**Reply by JessicaS11** [reply]
Woohoo! Thanks, @asteiker! Anyone know if AGU still sends emails to all the coauthors? I got several (initiation, payment, submission) for the abstracts I submitted, but wasn't sure which went to coauthors.

**Reply by danielfromearth** [reply]
Thanks @asteiker!  And @JessicaS11, I haven't received any notification yet either.

**Reply by mfisher87** [reply]
Awesome work :)

**Reply by asteiker** [reply]
@JessicaS11 I also haven't received any notifications as co-author. I don't think these are being sent this year, whether that's intentional or not.

**Reply by JessicaS11** [reply]
On the bright side, it cuts down on email clutter...
